crates/accelerate/src/sampled_exp_val.rs

```diff
@@ -53,6 +67,35 @@ fn bitstring_expval(dist: &HashMap<String, f64>, mut oper_str: String) -> f64 {
     exp_val / denom
 }
 
+fn bitstring_expval_bitterm(dist: &HashMap<String, f64>, term: &SparseTermView) -> PyResult<f64> {
+    let inds: Vec<usize> = term.indices.iter().map(|x| *x as usize).collect();
+    let n = term.num_qubits as usize;
+    let denom: f64 = fast_sum(&dist.values().copied().collect::<Vec<f64>>());
+    let exp_val: PyResult<f64> = dist
+        .iter()
+        .map(|(bits, val)| {
+            let temp_product: PyResult<f64> =
+                term.bit_terms
+                    .iter()
+                    .enumerate()
+                    .try_fold(1.0, |acc, (idx, bitterm)| {
+                        if (*bitterm as usize) & 0b0011 != 0b0001 {
```

Comment:
We don't need to interpret `bitterm` as `usize` here, do we?
```suggestion
                        if *bitterm & 0b11 != 0b01 {
```

Reply:
We need to it appears. It did not work for me.
<img width="1188" height="272" alt="image" src="https://github.com/user-attachments/assets/a419cc94-5b3c-4c0f-a65b-76c88fc2a6cb" />